k2/csrc/intersect_test.cu

```diff
@@ -30,8 +46,8 @@ TEST(Intersect, Simple) {
   // clang-format off
   DenseFsaVec dfsavec {
     RaggedShape("[ [ x x x ] ]"),
-    Array2<float>("[ [ -Inf 0.1 0.2 0.3 ] [ -Inf 0.04 0.05 0.06 ] [ 1.0 -Inf -Inf -Inf]]")  // NOLINT
-  };
+        Array2<float>("[ [ -Inf 0.1 0.2 0.3 ] [ -Inf 0.04 0.05 0.06 ] [ 1.0 -Inf -Inf -Inf]]")  // NOLINT
```

Comment:
Should the arc entering the final state have `score == 0` instead of 1.0 ?

Reply:
There isn't a special constraint here (it would have that value, though, if it came from a normal log-like matrix; I wanted to test something though).